packages/block-editor/src/components/block-list/insertion-point.js

```diff
@@ -84,6 +89,25 @@ export default function InsertionPoint( {
 		setInserterClientId( clientId );
 	}
 
+	function onClick( event ) {
```

Comment:
For these sorts of event handlers, I'd maybe suggest to name the associated function something more meaningful to describe what we're expecting to happen as a result of the click, as a limited form of self-documentation.

e.g. `focusClosestTabbable`